src/relay.js

```diff
@@ -171,13 +179,12 @@ export function sequelizeConnection({name, nodeType, target, orderBy: orderByEnu
     return result;
   };
 
-  let resolveEdge = function (item, args = {}, source) {
-    if (!args.orderBy) {
-      args.orderBy = [defaultOrderBy];
-    }
-
+  let resolveEdge = function (item, index, cursor, args = {}, source) {
+    let startIndex = 0;
+    if (cursor) startIndex = Number(cursor.index);
+    if (startIndex !== 0) startIndex++;
```

Comment:
I went ahead and renamed this to be clearer. I am holding off on committing until I can make any final changes we may need.